tests/test1020.py

```diff
@@ -0,0 +1,24 @@
+#Test getter and setter of font.style_set_names
```

Comment:
As noted or implied above, a bunch more tests for error conditions and sanity when changing the list multiple times would be good here.